File: MPI/mpicheck/tests/functions/2.2/MPI_Dist_graph_neighbors_count.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Comm var_0;
    int *var_1;
    int *var_2;
    int *var_3;
    int ret;
    /* calls */
    ret = MPI_Dist_graph_neighbors_count(var_0, var_1, var_2, var_3);
    ret = PMPI_Dist_graph_neighbors_count(var_0, var_1, var_2, var_3);
    return 0;
}

File: MPI/mpicheck/tests/functions/2.2/MPI_Dist_graph_neighbors_count.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2
       LOGICAL var_3
       INTEGER var_4
        call mpi_dist_graph_neighbors_count(var_0, var_1, var_2, var_3, var_4)
       call pmpi_dist_graph_neighbors_count(var_0, var_1, var_2, var_3, var_4)
        end program main
    